aptos-move/framework/aptos-framework/sources/aptos_governance.move

```diff
@@ -359,7 +369,7 @@ module aptos_framework::aptos_governance {
             execution_hash,
             metadata_location,
             metadata_hash,
-            is_multi_step_proposal
+            is_multi_step_proposal,
```

Comment:
What's the thought on having vs not having trailing commas?

Reply:
This is just merely imitating rustfmt; personally, I prefer to always remove the trailing comma in the list.